geopandas/geodataframe.py

```diff
@@ -698,6 +698,14 @@ def __setitem__(self, key, value):
     # Implement pandas methods
     #
 
+    def drop(self, labels, axis=0, level=None, inplace=False, errors='raise'):
+        result = super(GeoDataFrame, self).drop(labels, axis=axis, level=level,
+                                                inplace=inplace, errors=errors)
```

Comment:
```suggestion
    def drop(
        self,
        labels=None,
        axis=0,
        index=None,
        columns=None,
        level=None,
        inplace=False,
        errors="raise",
    ):
        result = super(GeoDataFrame, self).drop(
            labels=labels,
            axis=axis,
            index=index,
            columns=columns,
            level=level,
            inplace=inplace,
            errors=errors,
        )
```

Update to mirror current pandas API (blacken).